launchable/commands/split_subset.py

```diff
@@ -81,8 +95,73 @@ def run(self):
                 payload = {
                     "sliceCount": count,
                     "sliceIndex": index,
+                    "sameBins": [],
                 }
 
+                tests_in_files = []
+
+                if same_bin_files is not None:
+                    same_bins = []
+                    for same_bin_file in same_bin_files:
+                        with open(same_bin_file, "r") as f:
+                            """
+                            A same_bin_file expects to have a list of tests with one test per line.
+                            Each line of test gets formatted and packed to sameBins list in payload.
+                            E.g.
+                                For gradle:
+                                ```
+                                $ cat same_bin_file.txt
+                                example.AddTest
+                                example.DivTest
+                                example.SubTest
+                                ```
+                                Formatted:
+                                ```
+                                "sameBins" [
+                                    [
+                                        [{"class": "example.AddTest"}],
+                                        [{"class": "example.DivTest"}],
+                                        [{"class": "example.SubTest"}]
+                                    ]
+                                ]
+                                ```
+                            E.g.
+                                For gotest:
+                                ```
+                                $ cat same_bin_file.txt
+                                example.BenchmarkGreeting
+                                example.ExampleGreeting
+                                ```
+                                Formatted:
+                                ```
+                                "sameBins" [
+                                    [
+                                        [
+                                            {"class": "example"}, 
+                                            {"testcase": "BenchmarkGreeting"}
+                                        ],
+                                        [
+                                            {"class": "example"}, 
+                                            {"testcase": "ExampleGreeting"}    
+                                        ]
+                                    ]
+                                ]
+                                ```
+                            """
+                            t = f.readlines()
```

Comment:
Can you make these variables a little more understandable? I'm a little confused with t, s, u...

Reply:
Fixed.
https://github.com/launchableinc/cli/pull/470/commits/fa3e13adae5387146120bf062b592a79f7e7ba6d